### PR TITLE
Introduce `DISCONNECTED` connection quality.

### PR DESCRIPTION
**Pull request by boks1971**
Currently, this state happens when any up stream track does not send any packets in an analysis window when it is expected to send packets.

This can be used by participants to know the quality of a potentially disconnected participant. Previously, it took 20 - 30 seconds for the stale timeout to kick in and disconnect the limbo participant which triggered a participant update through which other participants knew about it.

Previously, `POOR` quality was also overloaded to denote that the up stream is not sending any packets. With this change, that is a separate indicator, i. e. `DISCONNECTED`.

For https://github.com/livekit/livekit/issues/2048

### CODE REVIEW

**Review by cnderrauber — APPROVED** [review]
Looks good!

**Review by paulwe — APPROVED** [review]

File: pkg/sfu/connectionquality/scorer.go [hunk 1]
```diff
@@ -531,7 +536,11 @@ func scoreToConnectionQuality(score float64) livekit.ConnectionQuality {
 		return livekit.ConnectionQuality_GOOD
 	}
 
-	return livekit.ConnectionQuality_POOR
+	if score > 20.0 {
```
Comment (davidzhao) [inline pkg/sfu/connectionquality/scorer.go:539]: nit: better to use const `cMinScore` here?

Comment (boks1971) [reply pkg/sfu/connectionquality/scorer.go:539]: Was just using numbers there. Will change it. Actually, can use 'cDisconnectedScore' which might be easier to read. Will make constants for others too in a separate PR.

**Comment by arnasledev** [comment]
hi, when do you plan to release this?

**Pull request merged**